src/sql/base/browser/ui/panel/media/panel.css

```diff
@@ -149,7 +147,50 @@ panel {
 	font-weight: bold;
 	margin: 15px 5px 0px 5px;
 	line-height: 35px;
+	height: 35px;
 	border-style: solid;
 	border-width: 0 0 1px 0;
 	border-color: rgb(214, 214, 214);
 }
+
+.tabbedPanel .tab-group-heade {
```

Comment:
>heade [](start = 24, length = 5)

missing r?

Reply:
I have removed this. since it is not needed, since block is the default value for display.